dev.docker-compose.yml

```diff
@@ -68,6 +68,7 @@ services:
             - ./volume/properties/SystemConfiguration.properties:/var/lib/openelis-global/properties/SystemConfiguration.properties
             -  lucene_index-vol:/var/lib/lucene_index
             - ./volume/test-map/test-loinc-map.csv:/var/lib/openelis-global/plugin-test-mappings/test-loinc-map.csv
+            - ./src/main/resources/configurations/ocl:/usr/local/tomcat/configurations/ocl
```

Comment:
@vsvishalsharma this should be loaded for the external file not on the classpath.
ie under ` ./volume/configurations/ocl:/var/lib/openelis-global/configurations/ocl`
That way some one shouldnt have to compile a new war file to load a new zip